grc/templates/upload/_upload-files-section.html

```diff
@@ -32,7 +31,13 @@ <h2 class="govuk-heading-m">
 
                     <div class="govuk-summary-list__row">
                         <dd class="govuk-summary-list__value govuk-!-width-full" style="vertical-align: middle;">
+                            {% if evidence_file.password_required %}
+                            <img src="{{ url_for('static', filename='assets/images/padlock.svg') }}" style="width: 30px; vertical-align: bottom; margin-right: 2px;">
+                            {% endif %}
+
+                            {# <a href="{{ url_for('upload.download', section_url=section_url) }}?file={{ evidence_file.aws_file_name }}" class="govuk-link"> #}

```

Comment:
This code is commented-out.

I'm happy for you to keep it on a branch (in case we want it later), but if we merge it into the main branch, it will be confusing for us and future developers